superset-frontend/src/components/Select/Select.tsx

```diff
@@ -385,31 +385,37 @@ const Select = (
 
   const hasCustomLabels = fullSelectOptions.some(opt => !!opt?.customLabel);
 
+  const valueAsArray = <T,>(value: any): T[] => {
+    const array = value ? (Array.isArray(value) ? value : [value]) : [];
+    return array as T[];
+  };
+
   const handleOnSelect = (
     selectedValue: string | number | AntdLabeledValue,
```

Comment:
Can we rename this local `selectedValue` to `selectedItem` so not to confuse with `selectValue` in the parent context which may also be an array?